components/benefit_sponsors/app/controllers/benefit_sponsors/benefit_packages/benefit_packages_controller.rb

```diff
@@ -2,12 +2,14 @@ module BenefitSponsors
   module BenefitPackages
     class BenefitPackagesController < ApplicationController
 
+      before_action :check_for_late_rates, only: [:new]
+
       include Pundit
 
       layout "two_column"
 
       def new
-        @benefit_package_form = BenefitSponsors::Forms::BenefitPackageForm.for_new(params.require(:benefit_application_id))
+        @benefit_package_form ||= BenefitSponsors::Forms::BenefitPackageForm.for_new(params.require(:benefit_application_id))
```

Comment:
We are already initializing the same form in private method, please remove this.